src/Kunstmaan/GeneratorBundle/Resources/SensioGeneratorBundle/skeleton/defaultsite/DataFixtures/ORM/DefaultSiteGenerator/DefaultSiteFixtures.php

```diff
@@ -445,176 +456,192 @@ private function createContentPages()
-            $translation->setTitle('Climate research satellites');
-            $translation->setSlug('climate-research-satellites');
-        });
-        $translations[] = array('language' => 'nl', 'callback' => function($page, $translation, $seo) {
-            $translation->setTitle('Klimatologische onderzoekssatellieten');
-            $translation->setSlug('klimatologische-onderzoekssatellieten');
-        });
-
-        $options = array(
-            'parent' => $satellitePage,
-            'page_internal_name' => 'climate-research-satellites',
-            'set_online' => true,
-            'hidden_from_nav' => false,
-            'creator' => self::ADMIN_USERNAME
-        );
-
-        $this->pageCreator->createPage($satelliteOverviewPage, $translations, $options);
-
-        $satelliteOverviewPage = new SatelliteOverviewPage();
-        $satelliteOverviewPage->setTitle('Passive satellites');
-        $satelliteOverviewPage->setType(Satellite::TYPE_PASSIVE);
-
-        $translations = array();
-        $translations[] = array('language' => 'en', 'callback' => function($page, $translation, $seo) {
-            $translation->setTitle('Passive satellites');
-            $translation->setSlug('passive-satellites');
-        });
-        $translations[] = array('language' => 'nl', 'callback' => function($page, $translation, $seo) {
-            $translation->setTitle('Passieve satellieten');
-            $translation->setSlug('passieve-satellieten');
-        });
-
-        $options = array(
-            'parent' => $satellitePage,
-            'page_internal_name' => 'passive-satellites',
-            'set_online' => true,
-            'hidden_from_nav' => false,
-            'creator' => self::ADMIN_USERNAME
-        );
-
-        $this->pageCreator->createPage($satelliteOverviewPage, $translations, $options);
-
-        $list = array(
-            array('Sputnik 1', '1957-10-04', 'http://en.wikipedia.org/wiki/Sputnik_1', 84, Satellite::TYPE_COMMUNICATION),
-            array('Echo 1', '1960-08-12', 'http://en.wikipedia.org/wiki/Echo_satellite', 180, Satellite::TYPE_COMMUNICATION),
-            array('Telstar 1', '1962-07-10', 'http://en.wikipedia.org/wiki/Telstar', 70, Satellite::TYPE_COMMUNICATION),
-            array('Intelsat I', '1965-04-06', 'http://en.wikipedia.org/wiki/Intelsat_I', 149, Satellite::TYPE_COMMUNICATION),
-
-            array('ACRIMSAT', '1999-12-20', 'http://en.wikipedia.org/wiki/ACRIMSAT', 288, Satellite::TYPE_CLIMATE),
-            array('Terra', '1999-12-18', 'http://en.wikipedia.org/wiki/Terra_(satellite)', 4864, Satellite::TYPE_CLIMATE),
-            array('GRACE', '2002-03-14', 'http://en.wikipedia.org/wiki/Gravity_Recovery_and_Climate_Experiment', 487, Satellite::TYPE_CLIMATE),
-            array('Landsat 7', '1999-04-15', 'http://en.wikipedia.org/wiki/Landsat-7', 1973, Satellite::TYPE_CLIMATE),
-            array('SORCE', '2003-01-25', 'http://en.wikipedia.org/wiki/SORCE', 315, Satellite::TYPE_CLIMATE),
+{
```

Comment:
can you revert this code formatting? looks like this was not needed, and it makes reviewing PR's hard :-)